### PR TITLE
Add ghost mode button and game button to joystick

### PR DESCRIPTION
**Pull request by ktpss95112**

### CODE REVIEW

File: sites/game-client/body-mobile.ejs [hunk 1]
```diff
@@ -61,7 +61,11 @@
           <canvas id="inputcanvas"></canvas>
           <div id="joystick-container">
             <!-- TODO: styling -->
-            <div id="joystick-control-point"></div>
+            <div id="joystick-control-point-container">
+              <div id="joystick-control-point"></div>
+            </div>
+            <div id="joystick-ghost-mode-button" class="container-center hoverable"><img src="/static/sites/game-client/input-manager/ghostmode.png"></div>
+            <div id="joystick-game-button" class="container-center hoverable"><img src="/static/sites/game-client/input-manager/bomb.png"></div>
```
Comment (john0312) [inline sites/game-client/body-mobile.ejs:68]: Not really sure if we should make this part of the bombman extension, as in, if we remove the bombman extension, the button will disappear. If it takes too much time, we can leave a TODO here instead.

File: sites/game-client/input-manager/input-manager.mjs [hunk 2]
```diff
@@ -319,37 +346,83 @@ class InputManager {
...
-   * @param {Element} containerDiv - The container of the joystick.
+   * @param {InputManager} inputManager - The input manager which this joystick belongs to.
+   * @param {Element} containerDiv - The outer container of the joystick.
+   * @param {Element} controlPointContainerDiv - The container of the joystick control point.
    * @param {Element} controlPointDiv - The div to display the stick of the joystick.
+   * @param {Element} ghostModeDiv - The div to switch on/off ghost mode.
+   * @param {Element} gameButtonDiv - The div used by game (usually by extensions).
    */
-  constructor(containerDiv, controlPointDiv) {
+  constructor(inputManager, containerDiv, controlPointContainerDiv, controlPointDiv, ghostModeDiv, gameButtonDiv) {
```
Comment (john0312) [inline sites/game-client/input-manager/input-manager.mjs:356]: Not really sure if we should hard code the list of buttons we've, but if making it flexible takes too much time, then we can consider adding a TODO here.

**Review by john0312 — APPROVED** [review]

File: sites/game-client/body-mobile.ejs (hunk 1 above)
Comment (ktpss95112) [reply sites/game-client/body-mobile.ejs:68]: The "game button" would be used by any extension which needs a button. It is not limited to bombman extension. For example, battle royale may need to use this button as well.
It requires too much engineering to allow extensions to add buttons. Let it be TODO would be good for now.

File: sites/game-client/input-manager/input-manager.mjs (hunk 2 above)
Comment (ktpss95112) [reply sites/game-client/input-manager/input-manager.mjs:356]: Answered above.

**Review by john0312 — APPROVED** [review]
I'm fine with adding a TODO, still LGTM.

**Pull request merged**